### ISSUE TITLE
API return invalid csrf token

### ISSUE THREAD

**Issue by haluklevent** [openstf/stf#439]
Hi,

I am developing new app with STF API but when i post 
url: /auth/api/v1/mock 
data : {"name":"test", "email":"test@gmail.com"} 

over postman, API return "invalid csrf token" error. I guess csrf can go in session. What i can do for it or must i change the authentication strategy over API ?

**Comment by codeskyblue** [comment]
You should use token for authentication

**Comment by haluklevent** [comment]
@codeskyblue how can i do with Postman ? I know, i tired you but i need help :(

**Comment by codeskyblue** [comment]
login must be manual

**Comment by haluklevent** [comment]
this is not usefull API, thank you so much

**Comment by sorccu** [comment]
I don't think you understand how the API works... you login once and create
an API token in the UI. You can then use the API methods with that token
(i.e. as that user). You definitely do NOT send anything to the login
endpoint.

**Comment by haluklevent** [comment]
@sorccu you are right man but i dont want to login on browser screen, my web web application use that API step by step login>devices list>device details and test.

A lot user(recorded in rethink) can uses STF over my web application, how can i generate different  token for each person? Because they want to use same device and i cannot handle this situation.
